source/developers/plugins/index.rst

```diff
@@ -0,0 +1,38 @@
+:is-up-to-date: True
+
+.. index:: Crafter CMS Plugins
+
+=======
+Plugins
+=======
+
+
+Crafter CMS plugins extend the CMS and provide the user additional functionality.  Plugins can be a single file or multiple files and directories.
+
+Here are the available plugin types:
+
+* Blueprints
+* Studio Plugins
+
+Crafter CMS currently has a `marketplace <https://marketplace.craftercms.org>`__ for users to submit their plugins for others to use or to find plugins that others have created. The Crafter Marketplace currently only supports the blueprint plugin type.  See :ref:`marketplace` for more information.
+
+**Blueprints**
+
+Blueprints provide an initial structure for a site.  The following contains details on how to create a blueprint that you can submit to the Crafter Marketplace:
```

Comment:
Blueprints provide an initial set of styles, pages, behaviours, content types and more. An "initial structure" feels a little shy of what they are.